### ISSUE TITLE
Error 500 when viewing all pull requests

### ISSUE THREAD

**Issue by simonkulessa** [mendhak/bitbucket-all-pull-requests#3]
Hi,

Bitbucket Version: 5.16.2 (i know, its old)
Plugin Version: 1.9 (latest release here)

Error from Log File:

2021-05-04 08:29:40,239 DEBUG [http-nio-7990-exec-4] s.kulessa @BVYVAUx509x102136x1 17wvg1n 192.168.xxx.xxx,0:0:0:0:0:0:0:1 "GET /plugins/servlet/all-pull-requests/all HTTP/1.1" c.a.s.i.i18n.PluginI18nService No values found in any valid locale for key mergeabilityColumnHeader and locales [de_DE, de, en_US, en]
2021-05-04 08:29:40,244 ERROR [http-nio-7990-exec-4] @BVYVAUx509x102143x1 17wvg1n 192.168.xxx.xxx,0:0:0:0:0:0:0:1 "GET /mvc/error500 HTTP/1.1" c.a.s.i.web.ErrorPageController There was an unhandled exception loading [/plugins/servlet/all-pull-requests/all]
com.google.template.soy.tofu.SoyTofuException: Attempting to render undefined template 'bitbucket.internal.feature.project.projectAvatar.projectAvatar.avatar'.

So right now I dont know what to do, I always get that 500 error.

Any ideas?

Best regards
Simon

**Comment by mendhak** [comment]
Yes I think this won't work with BB5.  The reason is, the original plugin was written for BB5, but didn't work in BB6 onwards.  

As part of the BB6 'fix', there were many breaking template changes, including a line about the project avatar, which is what your error is pointing at:

https://github.com/mendhak/bitbucket-all-pull-requests/commit/256fe0911511cd583855b9b2dabcb317fb2de6c3#diff-aa735977faedbbba3dc642085fa1a693da2e3217b86682dd29bb8bb87fa31e4dL88

So for your setup you'll want the original plugin: 

https://bitbucket.org/infusiondev/stash-all-pull-requests-extra/src/master/

**Comment by simonkulessa** [comment]
Ah ok, 
I see, there seems to be no release, have to build it myself? :D

Thanks and regards
Simon

**Comment by mendhak** [comment]
You can download it here:
https://bitbucket.org/infusiondev/stash-all-pull-requests-extra/downloads/

But it should still be available in the market place, isn't it there?

**Comment by simonkulessa** [comment]
Unfortunately, no.

Seems to be the old release, not the new one with the patches for 5 (from 2017), I'll read a bit about compiling it and hopefully it'll work. 

Thank you :)
